components/Navbar/Tab.tsx

```diff
@@ -32,6 +32,7 @@ export default function Tab({
 
   const challengeId = isRouteLesson ? pathData.pop().split('-')[0] : undefined
   const isActive = challenge.lessonId === challengeId
+  const isLast = index == count - 1
```

Comment:
Let's use `===` to make sure JS doesn't make any assumptions for us. Should work like this, but its a good practice to use `===` whenever you can.

Reply:
Sure! Let me fix this. Always avoid assumptions! ✌️